e2e/test/scenarios/admin-2/error-reporting.cy.spec.ts

```diff
@@ -42,6 +45,25 @@ describe("error reporting modal", () => {
     });
   });
 
+  it("should allow you to open the error reporting modal via the command palette", () => {
+    restore();
+    cy.signInAsAdmin();
+    cy.visit("/");
+
+    cy.findByTestId("home-page")
+      .findByText(/see what metabase can do/i)
+      .realClick();
+    cy.wait(500);
```

Comment:
wait bad. at least put a comment on why it is needed.

Reply:
I was able to remove it, this was just copied from the test above it. Basically you need to give the app time to register listeners before you try to use keyboard shortcuts (I faced something similar when writing command palette tests). I transitioned to opening the command palette via the search button instead